ci-operator/step-registry/gcp/bastion/post/gcp-bastion-post-ref.yaml

```
@@ -0,0 +1,13 @@
+ref:
+  as: gcp-bastion-post
+  from_image:
+    namespace: ocp
+    name: "4.5"
```

```suggestion
    name: "4.10"
```
Lets use latest installer which most likely has 4.5+ deprovision issue solved (if any)